packages/fast-components-react-base/src/number-field/number-field.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { get } from "lodash-es";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import {
+    NumberFieldHandledProps,
+    NumberFieldManagedClasses,
+    NumberFieldProps,
+    NumberFieldUnhandledProps,
+} from "./number-field.props";
+import {
+    ManagedClasses,
+    NumberFieldClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-base";
+import { TextFieldType } from "../text-field/index";
+
+class NumberField extends Foundation<
+    NumberFieldHandledProps,
+    NumberFieldUnhandledProps,
+    {}
+> {
+    public static defaultProps: Partial<NumberFieldProps> = {};
+
+    public static displayName: string = "NumberField";
+
+    protected handledProps: HandledProps<NumberFieldHandledProps> = {
+        managedClasses: void 0,
+        value: void 0,

```

Comment:
@janechu and @nalogiudice can we jump on discord and sync up sometime today about the handled prop contract here?

Reply:
I think it will only be `managedClasses`. @janechu has specified all props can be passed via unhandled props. If we need a chat that's fine, or you may just be able to talk to Jane.